src/pages/validator/amplifier/governance-proposals.mdx

```diff
@@ -38,13 +40,13 @@ axelard tx gov submit-proposal execute-contract $SERVICE_REGISTRY_ADDRESS \
             "verifiers":[
                 "'"$VERIFIER_AXELAR_ADDRESS"'"
             ],
-            "service_name":"validators"
+            "service_name":"'"$SERVICE_NAME"'"
         }
     }' \
-    --title="Change [chain name]'s verifier set" \
-    --description="Change [chain name]'s verifier set" \
+    --title="Change "'"$CHAIN_NAME"'"'s verifier set" \
```

Comment:
This isn't right. The service name will be given to folks (right now it's `validators`) and will be universal. This isn't adding a verifier to a chain, this is adding a verifier to Amplifier (all verifiers are approved at the amplifier level and the individual verifiers decide which chain to support without governance change.